src/main/routes.ts

```diff
@@ -80,7 +81,14 @@ export class Routes {
         if (!req.session.user) {
           return res.redirect(SIGN_OUT_URL);
         }
-        return res.end();
+        req.session.cookie.expires = new Date(Date.now() + cookieMaxAge);
+        req.session.cookie.maxAge = cookieMaxAge;
```

Comment:
Would it be better to have these 2 options set/moved into `src/main/modules/session/index.ts:26` or does that cause issues?

Reply:
hmmm, maybe, the idea was to have it here as active was meant to extend the session when the user was active.